policyDefinitions/App Configuration/app-configuration-stores-should-should-have-soft-delete-enabled-of-7-days/azurepolicy.rules.json

```diff
@@ -0,0 +1,31 @@
+{
+  "if": {
+    "allOf": [
+      {
+        "field": "type",
+        "equals": "Microsoft.AppConfiguration/configurationStores"
+      },
+      {
+        "not": {
+          "field": "Microsoft.AppConfiguration/configurationStores/createMode",
+          "equals": "recover"
+        }
+      },
+      {
+        "anyOf": [
+          {
+            "field": "Microsoft.AppConfiguration/configurationStores/softDeleteRetentionInDays",
+            "exists": "false"
+          },
+          {
+            "field": "Microsoft.AppConfiguration/configurationStores/softDeleteRetentionInDays",
+            "notequals": 7
```

Comment:
Same as other comment, but just a reminder that it would need to be changed here as well.